### PR TITLE
ROC-3458: no cache added

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-3458

### Change description
No cache

### Work checklist

- [x] Unit tests added where applicable
- [x] Route tests added for new pages
- [x] New pages included in a11y tests
- [x] Task list and task completeness checks updated
- [x] Check and send page updated
- [x] Routes, page content and page flows of new features are toggled 
- [x] UI changes look good on mobile
- [x] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/app.ts [hunk 1]
```diff
@@ -38,6 +39,7 @@ app.locals.ENV = env
 
 const developmentMode = env === 'development'
 
+app.use(noCache())
```
Comment (geek-soft) [inline src/main/app.ts]: Please use `app.use(helmet.noCache())` instead. Helmet also brings `nocache` dependency, no need to add it manually.

Comment (geek-soft) [reply src/main/app.ts]: Also best to add this to `Helmet.enableFor` in `modules/helmet/index.ts`.

Comment (yarpo) [reply src/main/app.ts]: ok

File: yarn.lock [hunk 2]
```diff
@@ -5357,7 +5357,7 @@ nise@^1.2.0:
     path-to-regexp "^1.7.0"
     text-encoding "^0.6.4"
 
-nocache@2.0.0:
+nocache@^2.0.0:
```
Comment (geek-soft) [inline yarn.lock]: Can you please revert that change?

File: src/main/modules/helmet/index.ts [hunk 3]
```diff
@@ -19,6 +19,7 @@ export class Helmet {
 
   enableFor (app: express.Express) {
     app.use(helmet())
+    app.all(/^.+(?!\/public\/.*)$/, helmet.noCache())
```
Comment (geek-soft) [inline src/main/modules/helmet/index.ts]: I suggest `^\/(?!public).*$`. Your regex matches `/public/stylesheets/application.css` which means that that file would not get cached.

**Pull request merged**